src/Sonarr.Api.V3/Config/UiConfigController.cs

```diff
@@ -16,6 +17,8 @@ public UiConfigController(IConfigFileProvider configFileProvider, IConfigService
             : base(configService)
         {
             _configFileProvider = configFileProvider;
+            SharedValidator.RuleFor(c => c.UILanguage)
```

Comment:
Could do something like what we do in `LanguageSpecification` to make it a little more strict, though I'd also add in a check to make sure the value is greater than or equal to 1 with a custom message.